docs/src/_includes/components/nav-version-switcher.html

```diff
@@ -1,7 +1,8 @@
 <div class="version-switcher">
-    <a href="/versions/" class="switcher-fallback">Versions</a>
-
-    <div hidden role="region" class="switcher switcher--version" aria-labelledby="nav-version-switcher-label" id="nav-version-switcher">
+    <noscript>
+            <a href="/versions/" class="switcher-fallback">Versions</a>
+    </noscript> 
```

Comment:
Could we use the `enhanced` class name (the one that replaces `no-js` on html) to hide all fallback links, and then we won't need to use `<noscript>`?

Something like `.enhanced .switcher-fallback { display: none !important }` in the `<style>` we added to `<head>`.

Reply:
yes, we can use the `enhanced` class instead of `<noscript>`.